examples/default/channels/inventory.events/index.md

```diff
@@ -0,0 +1,13 @@
+---
+id: inventory.events
+name: Inventory Events
+version: 1.0.1
+summary: |
+  The topic on which measured values may be produced and consumed.
+owners:
+    - dboyne
+address: smartylighting/streetlights/1/0/event/5/lighting/measured
+technology: kafka
```

Comment:
Per Asyncapi spec this should be "protocol" as it mustly refers to servers object practically 

Per Cloudevents this is called also protocol but behind a bindings concept https://github.com/cloudevents/spec/tree/main/cloudevents/bindings

Reply:
Thanks! totally makes sense, funny I actually changed it the other day when I was working on it and just seen the comment. Thanks for the review/comment! Changed to protocol